draft-looker-jwm.xml

```diff
@@ -28,9 +28,9 @@
     <date day="21" month="December" year="2019"/>
 
     <abstract>
-      <t>JSON Web Message (JWM) is a flexible way of describing an application-level message encoded in JSON that is suitable 
-      for transfer over a variety of transport protocols. JWMs through the application of JSON Web Signature (JWS) and JSON Web 
-      Encryption (JWE) can be digitally signed, integrity protected and or achieve confidentiality via encryption.</t>
+      <t>JSON Web Message (JWM) is a flexible way to encode application-level messages in JSON
+      for transfer over a variety of transport protocols. By using JWMs in conjunction with JSON Web Signature (JWS) and JSON Web
```

Comment:
I get tripped up on the usage of the word conjunction here, I still feel like JWMs "apply" JWS and JWE, did you have any further rationale that drove this word selection?

Reply:
"in conjunction" feels correct in this sense because a JWM isn't written to apply some other format like COSE as it stands. It works "together with" (the definition from merriam) and only with JWEs and JWSs.

Reply:
I removed "in conjunction" and replaced it with "use" (a JWM "uses" JWE and JWS). Is that better?